### DISCUSSION TITLE
Can kafka instance use external zookeeper cluster?

### DISCUSSION THREAD

**Discussion by tanberBro** [strimzi/strimzi-kafka-operator#9457] [Q&A] [answered]
Can kafka instance yaml set external zookeeper cluster in `kind: Kafka` file?
Or other method use external zookeeper cluster.

**Comment by ppatierno** [comment, marked as answer]
No it's not possible. The operator deploys its own ZooKeeper ensemble using it for the Kafka cluster. And it's also not accessible from any other application.

**Reply by tanberBro** [reply]
get it, thanks.

**Reply by ppatierno** [reply]
When someone answers your question, please use the "Mark as answer" button as well. It helps people to know what's the right answer to a common problem/question. Thanks!

**Reply by tanberBro** [reply]
done
